javascript/javascript1/week3/noteTakingApp.js

```diff
@@ -0,0 +1,38 @@
+const notes = [];
+
+function saveNote(content, id) {
+    if (notes.find(item => item.id === id)) {
+        console.log(`The Id you just entered ${id}, exists in notes please enter another ID`)
+    } else {
+        notes.push({ content: content, id: id })
+    }
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+
+function getNote(id) {
+    const note = notes.find(item => item.id === id)
```

Comment:
Is the variable `note` strictly necessary here? 🙂

Reply:
you mean something like this?
function getNote(id) {
    return notes.find(item => item.id === id);
}